system/modules/task/actions/edit.php

```diff
@@ -206,12 +209,13 @@ function edit_POST($w) {
     }
     
     $task->fill($_POST['edit']);
+    $task->rate = abs($task->rate) == 0 ? NULL : $task->rate;
```

Comment:
OK Is it really necessary? This line has changed in latest commit.

On Wed, Sep 14, 2016 at 1:16 PM, Adam Buckley <notifications@github.com>
wrote:

> In system/modules/task/actions/edit.php
> <https://github.com/2pisoftware/cmfive/pull/213#discussion_r78680788>:
>
> > @@ -206,12 +209,13 @@ function edit_POST($w) {
> >      }
> >
> >      $task->fill($_POST['edit']);
> > +    $task->rate = abs($task->rate) == 0 ? NULL : $task->rate;
>
> floatval is also a thing
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/2pisoftware/cmfive/pull/213/files/a0db7081daa98f7aea9915b52dc7151c39c34d60#r78680788>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIGsXP0uFTYXmFFRQCbyLx0QWI6CJ49bks5qp2cBgaJpZM4J8Rd5>
> .
>